content/documentation/getting-started/index.adoc

```diff
@@ -367,6 +367,38 @@ namespaces:
     - mycorp_test.*
 ----
 
+==== Namespace Selectors
+
+Kiali supports an optional label selector for namespaces which is used to fetch a subset of the available namespaces.
+
+The label selector is defined under the namespaces definition.
+
+The example below selects all namespaces that have a label `kiali-enabled: true`:
+
+[source,yaml]
+----
+namespaces:
+  label_selector: kiali-enabled=true
+----
+
+For further information how the `label_selector` interacts with `deployment.accessible_namespaces` read the https://github.com/kiali/kiali/blob/v1.1.0/operator/deploy/kiali/kiali_cr.yaml#L44-L53[technical documentation].
```

Comment:
minor:   "information how" => " information on how"

Also, I'd avoid versioning and line numbers in the link.  At the moment we don't have a versioning solution in the docs (we need that).  And line number can be brittle.  How about:

read the 'label_selector' documentation in https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml[kiali_cr.yaml].